testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/batch/repository/JobRepositoryTestUtils.java

```diff
@@ -39,7 +39,8 @@ public static void testGetJobExecutionsWithLimit(ServiceContainer serviceContain
             Assert.assertEquals(BatchStatus.COMPLETED, jobExecution.getBatchStatus());
         }
 
-        serviceContainer.dumpServices();
+        // TODO Current fails due to MSC-346
+        //serviceContainer.dumpServices();
```

Comment:
This commit seems unnecessary as WF now uses MSC 1.5.5 which should have this fixed.

Reply:
Great, I've dropped this commit.